components/src/ChatBot/chatbot.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import { Interactions } from 'aws-amplify';
+import { ThemeProvider } from 'styled-components';
+import ChatBot from 'react-simple-chatbot';
+import Review from './chatbot.review';
+
+import {getConfig, IEpConfig} from '../utils/ConfigProvider';
+
+let Config: IEpConfig | any = {};
+
+const botName = "EPConversationalInterface";
+
+const theme = {
+    background: '#f5f8fb',
```

Comment:
I'm wondering if we should clean this up by externalizing the styles into a chatbot.less file and include them here. what do you think?

Reply:
Great idea. I will implement this.